plugins/modules/docker_compose_v2.py

```diff
@@ -537,7 +537,7 @@ def get_up_cmd(self, dry_run, no_start=False):
         if dry_run:
             args.append('--dry-run')
         if self.yes:
-            args.append('--yes')
+            args.append('-y')
```

Comment:
I would add a comment here:
```suggestion
            # Note that the long form is '--y', and not '--yes' as one would expect. Since this looks like a bug we're using the short form.
            args.append('-y')
```